cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -775,6 +776,7 @@
       public static final String PROGRAM_KILLED_RUNS = "program.killed.runs";
       public static final String PROGRAM_REJECTED_RUNS = "program.rejected.runs";
       public static final String PROGRAM_NODE_MINUTES = "program.node.minutes";
+      public static final String PROGRAM_PROVISIONING_DELAY_SECONDS = "program.provisioning.delay.seconds";
```

Comment:
Is this only for unit test? If it is, define it in the test class

Reply:
It's not only for tests. This metric will be read by the metrics writer extension and published to stackdriver.